chrome/content/zotero/zoteroPane.xul

```diff
@@ -106,7 +106,19 @@
 					<toolbarbutton id="zotero-tb-library-add-menu" class="zotero-tb-button" tooltiptext="&zotero.toolbar.newLibrary.label;" type="menu">
 						<menupopup id="zotero-tb-library-add-popup">
 							<menuitem id="zotero-tb-group-add" label="&zotero.toolbar.newGroup;" oncommand="ZoteroPane_Local.newGroup()"/>
-							<menuitem id="zotero-tb-feed-add" label="&zotero.toolbar.newFeed.label;" command="cmd_zotero_newFeed"/>
+							<menu id="zotero-tb-feed-add-menu" label="&zotero.toolbar.feeds.new;">
```

Comment:
(In other words, optimize for the probably majority use case of people who don't know what a feed URL is or where to find one, but might find this feature useful if given an easy way to add feeds.)